crypto/txscript/test-data/script_tests-kip10.json

```diff
@@ -4240,53 +4204,53 @@
     "OP_RESERVED2 is reserved"
   ],
   [
-    "2147483648",
```

Comment:
Can you keep the below tests, just change the expected result? And write in the description something like "since kip10 we support 5 byte integers"

Reply:
commit 955424cd4c1dd72f4c1028b3bcdd3dd017bb6309